types/index.d.ts

```diff
@@ -0,0 +1,201 @@
+        isMarginTradingAllowed: boolean;
+        filters: Filter[];
+        permissions: Permission[];
+    }
+    export type ExchangeInformation = {
+        timezone: string;
+        serverTime: number; // Timestamp
+        rateLimits: RateLimit[];
+        exchangeFilters: Filter[];
+        symbols: SymbolInformation[];
+    }
+
+    // Trade
+    export type AccountInformation = {
+        makerCommission: number;
+        takerCommission: number;
+        buyerCommission: number;
+        sellerCommission: number;
+        canTrade: boolean;
+        canWithdraw: boolean;
+        canDeposit: boolean;
+        updateTime: number;
+        accountType: 'SPOT';
+        balances: {
+            asset: string;
+            free: string;
+            locked: string;
+        }[];
+        permissions: string[];
+    };
+    export type OrderSide = 'BUY' | 'SELL';
+    export type OrderType = 'LIMIT' | 'MARKET' | 'STOP_LOSS' | 'STOP_LOSS_LIMIT' | 'TAKE_PROFIT' | 'TAKE_PROFIT_LIMIT' | 'LIMIT_MAKER';
+    export type TimeInForce = 'GTC' | 'IOC' | 'FOK';
+    export type NewOrderRespType = 'ACK' | 'RESULT' | 'FULL';
+    export type OrderStatus = 'NEW' | 'PARTIALLY_FILLED' | 'FILLED' | 'CANCELED' | 'PENDING_CANCEL' | 'REJECTED' | 'EXPIRED';
+    export type OrderOptions = {
+        timeInForce?: TimeInForce;
+        quantity?: string | number;
+        quoteOrderQty?: string | number;
+        price?: string | number;
+        newClientOrderId?: string; // A unique id among open orders. Automatically generated if not sent.
+        stopPrice?: string | number; // Used with STOP_LOSS, STOP_LOSS_LIMIT, TAKE_PROFIT, and TAKE_PROFIT_LIMIT orders.
+        icebergQty?: string | number; // Used with LIMIT, STOP_LOSS_LIMIT, and TAKE_PROFIT_LIMIT to create an iceberg order.
+        newOrderRespType?: NewOrderRespType; // Set the response JSON. ACK, RESULT, or FULL; MARKET and LIMIT order types default to FULL, all other orders default to ACK.
+        recvWindow?: number; // The value cannot be greater than 60000
+    }
+    export type NewOrderResponseAck = {
+        symbol: string;
+        orderId: number;
+        orderListId: number; // Unless OCO, value will be -1
+        clientOrderId: string;
+        transactTime: number; // Timestamp
+    }
+    export type NewOrderResponseResult = NewOrderResponseAck & {
+        price: string;
+        origQty: string;
+        executedQty: string;
+        cummulativeQuoteQty: string;
+        status: OrderStatus;
+        timeInForce: TimeInForce;
+        type: OrderType;
+        side: OrderSide;
+    }
+    export type Fill = {
+        price: string;
+        qty: string;
+        commission: string;
+        commissionAsset: string;
+        tradeId: number;
+    };
+    export type NewOrderResponseFull = NewOrderResponseResult & {
+        fills: Fill[];
+    }
+
+    export class Spot {
+        constructor (apiKey ?: string, apiSecret?: string, options?: ClientOptions);
```

Comment:
I think you should add the `logger` property, so at least the user can run the example